<launch>
  <node pkg="rbx_vision" name="face_tracker" type="face_tracker.py" output="screen">

    <remap from="input_rgb_image" to="/camera/rgb/image_color" />
    <remap from="input_depth_image" to="/camera/depth/image" />
    
     <rosparam>
       show_text: True
       gf_maxCorners: 200
       gf_qualityLevel: 0.01
       gf_minDistance: 7
       gf_blockSize: 10
       gf_useHarrisDetector: False
       gf_k: 0.04
       feature_size: 2
       flip_image: False
    </rosparam>
    
  	<param name="cascade_frontal_alt" value="$(find rbx_vision)/data/haar_face_detector/haarcascade_frontalface_alt.xml" />
   	<param name="cascade_frontal_alt2" value="$(find rbx_vision)/data/haar_face_detector/haarcascade_frontalface_alt2.xml" />
  	<param name="cascade_profile" value="$(find rbx_vision)/data/haar_face_detector/haarcascade_profileface.xml" />
   	<param name="cascade_upperbody" value="$(find rbx_vision)/data/haar_face_detector/haarcascade_upperbody.xml" />
   	<param name="cascade_eye" value="$(find rbx_vision)/data/haar_face_detector/haarcascade_eye.xml" />

  </node>
</launch>
